FAQ/How to Write to an Existing Excel File.md

```diff
@@ -0,0 +1,33 @@
+# Write to an Existing Excel File
+### Enumerate the Excel File
+```powershell

```

Comment:
This is great. Could you add an opening sentence under the heading. Something along the lines of how `Open-ExcelPackage` and `Close-ExcelPackage` lets you work with the Excel contents directly. As compared to `Import-Excel` and `Export-Excel`. You don't need to do a comparison, just highlight the concept.

Reply:
Thanks for the feedback, that is a good idea! I have made this change.